js/checklist.js

```diff
@@ -0,0 +1,154 @@
+require(['jquery'], function($) {
+	/**
+	 * This adds a line of text after the "Import Content" <h1> on the content migration page
+	 * regarding migrating content from iSites. It's related to the modification to the
+	 * "Import Content" item on the setup checklist.
+	 *
+	 * NOTE: since the content of the "Import Content" form is dynamically generated by javascript
+	 * and the <h1> doesn't always exist in the DOM when this script is executed, the below code
+	 * polls for the existence of the element before trying to add the line of text. 
+	 */
+	var html = "<p>If you would like to incorporate content from a previous iSite, please contact the Academic Technology Group at <a href=\"mailto:atg@fas.harvard.edu\">atg@fas.harvard.edu</a>.</p>";
+	var is_content_migration_page = /^\/courses\/\d+\/content_migrations/.test(window.location.pathname);
+	var poll_for_element = pollForElement("#migrationConverterContainer > h1", 20, 100, function($el) {
+		$el.after(html);
+	});
+
+	if (is_content_migration_page) {
+		poll_for_element();
+	}
+
```

Comment:
Any line or lines in particular? The block comment at the top of the```require()``` block attempts to describe what the code does and why it's doing it that way. Are you looking for more of a line-by-line description?